.pending/breaking/sdk/4754-Switching-back-end-to-new-secret-store 

```diff
@@ -0,0 +1,10 @@
+#4754 Switching back-end to new secret store using library [Keyring](https://github.com/99designs/keyring)
```

Comment:
Sorry for any annoyances, but we've opted out of using a pending log now. Instead, simply update the current changelog under `Unreleased`.

Reply:
OMG

Reply:
ok, will update the changelog, no worries